tasks/pip.yml

```diff
@@ -58,7 +58,7 @@
 
 - name: Install uwsgi
   pip:
-    name: "{{ item }}"
+    name: http://http.debian.net/debian/pool/main/u/uwsgi/uwsgi_2.0.14+20170111.orig.tar.gz
```

Comment:
this approach is bad. `{{ item }}` represents a value that is set below.
You should determine this value conditionally, if it's debian stretch, insert this new string, otherwise keep the old package name.